integration-tests/src/test/scala/chiselTests/util/experimental/algorithm/Bitwise.scala

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: Apache-2.0
+
+import chisel3._
+import chisel3.util._
+import chisel3.experimental.util.algorithm._
+import chiseltest._
+import chiseltest.formal._
+import org.scalatest.flatspec.AnyFlatSpec
+import scala.math.min
+
+// Copied from rocket-core
+object RocketImpl {
+  // Fill 1s from low bits to high bits
```

Comment:
Can we use scaladoc comments so that these are well documented in IDEs?

Reply:
This is a test, no ScalaDoc needed

Reply:
oh nvm this is in the tests so probably not as important